src/libs/keystore/keystore.ts

```diff
@@ -179,6 +185,12 @@ export class Keystore {
     })
     // Persist the new secrets
     await this.storage.set('keystoreSecrets', secrets)
+
+    // produce uid if not exist (should created when first secret added)
```

Comment:
fix the english: "produce uid if one doesn't exist (should be created when the first secret is added)"